packages/ihe-gateway-sdk/src/client/ihe-gateway.ts

```diff
@@ -1,39 +1,25 @@
 import axios, { AxiosInstance } from "axios";
 import dayjs from "dayjs";
 import duration from "dayjs/plugin/duration";
-import { OutboundPatientDiscoveryReq } from "../models/patient-discovery/patient-discovery-requests";
 import { OutboundDocumentQueryReq } from "../models/document-query/document-query-requests";
 import { OutboundDocumentRetrievalReq } from "../models/document-retrieval/document-retrieval-requests";
+import { OutboundPatientDiscoveryReq } from "../models/patient-discovery/patient-discovery-requests";
 
 dayjs.extend(duration);
 
 const DEFAULT_AXIOS_TIMEOUT = dayjs.duration({ minutes: 2 });
 
-export enum APIMode {
-  dev = "dev",
-  integration = "integration",
-  production = "production",
-}
-
 export class IHEGateway {
-  static productionUrl = "https://ihe.metriport.com";
-  static integrationUrl = "https://ihe.staging.metriport.com";
-  static devUrl = "http://localhost:8082";
```

Comment:
It doesn't make sense to have these URLs, these are inbound URLs for external CQ gateways, and those call our inbound URLs using XCPD/XCA. This SDK is specific to use the IHE GW for outbound requests.

Reply:
yeah good point 👍